UI/Web/src/app/series-detail/series-detail.component.ts

```diff
@@ -247,16 +247,27 @@ export class SeriesDetailComponent implements OnInit, OnDestroy {
 	onScroll(): void {
 		const tabs = document.querySelector('.nav-tabs') as HTMLElement | null;
 		const main = document.querySelector('.main-container') as HTMLElement | null;
```

Comment:
Just to let you know, you don't need the as HtmlElement. You can also do an if (main === null) return;

It would never happen, but will allow you to not have to  use !.property.

Reply:
Resolved in https://github.com/Kareadita/Kavita/pull/1322/commits/f7ec0a67b1d0c0d34b5da56f8345cfd9d9614817